notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -406,3 +405,52 @@ fear [Bill Kenner admitted to Harold that "he was scared" fighting in the Great
 Christianity [Mrs. Krebs praised the Lord for having brought her son home from the war safe and sound. It is clear from the small town American context that this was the Christian God.]
 coping with a crisis of faith [While Harold's mother seems distressed that Harold cannot pray aloud with her, Harold seems very matter-of-fact about his unbelief.]
 
+
+theamericanshortstory10
+=======================
+
+:: Title
+The Golden Honeymoon
+
+:: Description
+"The Golden Honeymoon" is the ninth installment of The American Short Story
+television anthology series produced by Learning in Focus and Sea Cliff
+Productions for the Public Broadcasting Service. This installment is based on
+the 1922 Ring Lardner short story of the same name.
+
+Synopsis: An elderly couple go to Florida to celebrate their golden wedding
+anniversary.
+
+:: Date
+1980
+
+:: References
+https://www.imdb.com/title/tt0080804/
+
+:: Ratings
+4 <brent>
+4 <paul>
+
+:: Major Themes
+what it is like to travel [The story follows Charley and Lucy as they travel to St. Petersburg, Florida to celebrate their golden wedding anniversary. The viewer is shown what it was like for Charley and Cora to travel by train, stay at a hotel, and do things that old people stereotypically go to Florida to do.]
```

Comment:
Just to be clear: this theme applies to the active journeying part, and not activities undertaken at the destination.

Reply:
Could add a theme about "vacationing" which presumably would be a recreational activity..

Reply:
Good point. I've knocked it down to minor and revised the motivation accordingly.